src/content/games/h1/games.dot

```diff
@@ -102,8 +102,9 @@ digraph G {
   mac_204 -> halomd;
 
   # cea and mcc
-  pc_gold -> ann;
+  pc_gold -> ann [label="Campaign"];
   ann -> mcc_xbone;
+  pc_gold -> mcc_xbone [label="Multiplayer"];
```

Comment:
Adds a couple lines to the game evolution diagram showing that MCC came from CEA's MP and PC's campaign.

Reply:
I think it'd be more accurate to specify that MCC's campaign came from CEA, and CEA came from PC

Reply:
Updated...

like this?
![tmp-pr](https://user-images.githubusercontent.com/1519264/83210448-bc107d00-a10f-11ea-9647-66358b2a9b3d.png)